js/script.js

```diff
@@ -267,3 +267,31 @@ form.addEventListener('submit', (event) => {
     errorMsg.innerHTML = 'Please write email in small letters';
   }
 });
+
+// Preserve form data
```

Comment:
- [ ] Congratulations on the progress you have made! It appears that your data is successfully saved to localStore, however, after refreshing the browser, the data in the contact inputs are not present. It would be beneficial to review your code and ensure all desired functions are running optimally. Thank you for your hard work!

Reply:
Hi @Rustamxon7! Thanks for the valuable feedback. I have now fixed the issue to reload the existing data from the local storage.